project/Dependencies.scala

```diff
@@ -42,6 +42,7 @@ object Dependencies {
   val jodaConvert = "org.joda" % "joda-convert" % "1.2"
   val jSoup = "org.jsoup" % "jsoup" % "1.8.3"
   val liftJson = "net.liftweb" %% "lift-json" % "2.6-RC2"
+  val macwire = "com.softwaremill.macwire" %% "macros" % "2.2.3" % "provided"
```

Comment:
>  If you don't see a compilation error with Intellij that you see with sbt then Intellij is broken.

That wouldn't be the first time :smile: implicits and managed code (like the router) aren't handled nicely. (which is fine really, it's more of an edge case)

> I really don't get this point.

I'm saying as the needs of a component evolves and its dependencies change (all the time), it's nice to have the wiring done automatically, especially for components that have many dependencies, or repetitive dependencies. I will soon have to add `Environment` to pretty much every Controller of the application, and I would appreciate for my own sanity to have that done magically :smiley:.

> increase your compile time

I don't think it's significant enough to be a problem

> add accidental complexity

True, it's not a perfect solution

> that does not solve any problem (you should see any compilation error without it) 

It doesn't solve all wiring situations but it simplifies a whole bunch of it, which is an improvement in my opinion.

I'll give it a go without it for this PR, but if find myself in wiring hell during the following work I may re-introduce it.